tests/test-cases/Group6-VIC-Machine/6-16-Config.md

```diff
@@ -33,6 +33,8 @@ This test requires that a vSphere server is running and available
 20. Configure the VCH by attempting to change an existing container network
 21. Configure VCH http proxy
 22. Verify http proxy is set correctly through govc
+23. Configure VCH dns server to default 8.8.8.8 and 8.8.4.4
+24. Verify dns server is set correctly through govc
```

Comment:
inspect should work for dns, but if it doesn't, think that's a bug should be fixed in this issue. :)
let me know how you need any help.